src/Tween.ts

```diff
@@ -101,8 +101,13 @@ export class Tween<T extends UnknownProps> {
 
 		this._isChainStopped = false
 
-		this._startTime =
-			time !== undefined ? (typeof time === 'string' ? TWEEN.now() + parseFloat(time) : time) : TWEEN.now()
+		if (time === undefined) {
+			this._startTime = TWEEN.now()
+		}
+		else {
+			this._startTime = time
```

Comment:
What about `TWEEN.now() + time`?

Reply:
On second though, passing a string looks totally broken here. I'll fix this up and remove.